commit 6a97812013bf94538a4efc8b56d39848f9607977
Author: Eike Hein <hein@kde.org>
Date:   Tue Oct 29 12:13:08 2013 +0100

    Fix config state sync between desktop scripting and scripted applets.
    
    REVIEW:113447

diff --git a/plasma/configloader.cpp b/plasma/configloader.cpp
index 6e97cb9..ebc37fe 100644
--- a/plasma/configloader.cpp
+++ b/plasma/configloader.cpp
@@ -381,7 +381,7 @@ ConfigLoader::ConfigLoader(KSharedConfigPtr config, QIODevice *xml, QObject *par
 //       but KConfigSkeleton does not currently support this. it will eventually though,
 //       at which point this can be addressed properly
 ConfigLoader::ConfigLoader(const KConfigGroup *config, QIODevice *xml, QObject *parent)
-    : KConfigSkeleton(KSharedConfig::openConfig(config->config()->name()), parent),
+    : KConfigSkeleton(KSharedConfig::openConfig(config->config()->name(), KConfig::SimpleConfig), parent),
       d(new ConfigLoaderPrivate)
 {
     KConfigGroup group = config->parent();
